### PR TITLE
Shorten usage and remove options for now to allow regex to digest it.

### PR DESCRIPTION
**Pull request by aymanhab**
Fixes issue #2807

### Brief summary of changes
By trial and error, removed chunks of documentation parts to avoid regex exception at runtime on windows

### Testing I've completed
Ran tests, ran command in issue report live

### Looking for feedback on...
Is this enough documentation, what minimal additions would be enough for users

### CHANGELOG.md (choose one)

- no need to update because...
- updated.

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/2816)

### CODE REVIEW

File: Applications/opensim-cmd/opensim-cmd_viz.h [hunk 1]
```diff
@@ -60,18 +57,6 @@ Description of subcommands:
...
-  g, geometry  Search for geometry mesh files in this path.
-  m, model     If visualizing orientation data and --layout is 'model',
-               the orientation data will be visualized according to the default
-               pose of this model.
-  a, layout    If visualizing orientation data, this option specifies the layout
-               of x-y-z triads as either in a 'line' (the default), in a
-               'circle', or according to the default pose of a 'model'. If the
-               value is 'model', then the --model option must be provided.
-  r, rotate    If visualizing orientation data, this option specifies the space
-               fixed Euler angles to apply to data specified in radians.
```
Comment (chrisdembia) [inline Applications/opensim-cmd/opensim-cmd_viz.h:74]: Could you leave this documentation in the file, commented out, so we can add it back later?

Comment (aymanhab) [reply Applications/opensim-cmd/opensim-cmd_viz.h:74]: Absolutely, good idea 💡

File: Applications/opensim-cmd/opensim-cmd_viz.h [hunk 2]
```diff
@@ -60,6 +57,17 @@ Description of subcommands:
          TimeSeriesTableQuaternion, then the data are visualized as body
          orientations using x-y-z triads, and the --model and --layout
          flags can be used.
+Examples:
```
Comment (chrisdembia) [inline Applications/opensim-cmd/opensim-cmd_viz.h:60]: Please add a newline above this line.

Comment (chrisdembia) [inline Applications/opensim-cmd/opensim-cmd_viz.h:60]:
```suggestion

Examples:
```

**Comment by chrisdembia** [comment]
@aymanhab I added a newline. Feel free to merge

**Comment by aymanhab** [comment]
Awesome, thanks @chrisdembia

**Pull request merged**